UnitTests/UnitTests.XamlIslands.UWPApp/XamlIslandsTest_TokenizingTextBox.cs

```diff
@@ -0,0 +1,116 @@
+            new SampleDataType() { Text = "UnFavorite", Icon = Symbol.UnFavorite },
+            new SampleDataType() { Text = "UnPin", Icon = Symbol.UnPin },
+            new SampleDataType() { Text = "Zoom", Icon = Symbol.Zoom },
+            new SampleDataType() { Text = "ZoomIn", Icon = Symbol.ZoomIn },
+            new SampleDataType() { Text = "ZoomOut", Icon = Symbol.ZoomOut },
+        };
+
+        [TestInitialize]
+        public async Task Init()
+        {
+            await App.Dispatcher.ExecuteOnUIThreadAsync(() =>
+            {
+                _acv = new AdvancedCollectionView(_samples, false);
+
+                _acv.SortDescriptions.Add(new SortDescription(nameof(SampleDataType.Text), SortDirection.Ascending));
+
+                var xamlTokenizingTextBox = @"<controls:TokenizingTextBox xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation' 
+                                                 xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'
+                                                 xmlns:controls='using:Microsoft.Toolkit.Uwp.UI.Controls'
+                          PlaceholderText='Add Actions'
+                          MaxHeight='104'
+                          HorizontalAlignment='Center'
+                          VerticalAlignment='Center'
+                          TextMemberPath='Text'
+                          TokenDelimiter = ','>
+                        <controls:TokenizingTextBox.SuggestedItemTemplate>
+                          <DataTemplate>
+                            <StackPanel Orientation='Horizontal'>
+                              <SymbolIcon Symbol='{Binding Icon}'/>
+                              <TextBlock Text='{Binding Text}' Padding='4,0,0,0'/>
+                            </StackPanel>
+                          </DataTemplate>
+                        </controls:TokenizingTextBox.SuggestedItemTemplate>
+                        <controls:TokenizingTextBox.TokenItemTemplate>
+                          <DataTemplate>
+                            <StackPanel Orientation='Horizontal'>
+                              <SymbolIcon Symbol='{Binding Icon}'/>
+                              <TextBlock Text='{Binding Text}' Padding='4,0,0,0'/>
+                            </StackPanel>
+                          </DataTemplate>
+                        </controls:TokenizingTextBox.TokenItemTemplate>
+                      </controls:TokenizingTextBox>";
+
+                _tokenizingTextBox = XamlReader.Load(xamlTokenizingTextBox) as TokenizingTextBox;
+                _tokenizingTextBox.SuggestedItemsSource = _acv;
+
+                TestsPage.Instance.SetMainTestContent(_tokenizingTextBox);
+            });
+        }
+
+        [TestMethod]
+        public async Task TokenizingTextBox_PopupShowsInCorrectXamlRoot()
+        {
+            await App.Dispatcher.ExecuteOnUIThreadAsync(async () =>
+            {
+                await Task.Delay(10000);
```

Comment:
This test should probably try and add a token item and then select it with the keyboard to exercise the GetFocusedElement code, eh?

Reply:
I've added a much better/complete test for this:
https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3352/commits/39fe3849976e2ee5f2b48e38d0f99014b32ffd77